certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -347,9 +347,9 @@
     },
     "/accounts/{accountId}/envelopes/{envelopeId}/documents/{documentId}/tabs": {
       "get": {
-        "summary": "Get envelope document tabs",
-        "description": "Get envelope document tabs.",
-        "operationId": "GetEnvelopeDocumentTabs",
+        "summary": "Get envelope prefill tabs",
+        "description": "Get envelope prefill tabs.",
+        "operationId": "GetEnvelopePrefillTabs",
```

Comment:
This custom action returns all types of tabs(Prefilled and non-prefilled). Any reason you are restricting this to Prefill?

Reply:
For non-prefilled case, we have another action `get envelope recipient tabs` which you created

Reply:
Thinking about it, it makes sense to show both, as long as we provide a way to differentiate. See updated. Thanks for the comment.